src/dotnet/Core/Constants.cs

```diff
@@ -10,7 +10,7 @@ public static class Chat
         public static TileStack<Moment> TimeTileStack { get; } = TileStacks.Moment3MTo6Y;
         public static TimeSpan MaxEntryDuration { get; } = TimeTileStack.MinTileSize.EpochOffset; // 3 minutes, though it can be any
         public const int PictureFileSizeLimit = 10 * 1024 * 1024; // 10MB
-        public const int MaxRecentPeerChats = 5;
+        public const int MaxSearchCriteriaLength = 100;
```

Comment:
If I am not mistaken, I started to call it "search filter".

It's mostly because of two things:
- Shorter is better
- "Filter" is kinda less broad than "Criteria" (criteria can be arbitrary complex - i.e. all kinds of fields you search by, etc., filter is perceived as something that's simpler).

Reply:
done